Mp lxai

mpantoja314 opened the pull request:
Sorted poster presenters
added new sponsors
added link to registration for workshop

quickresolve commented on _includes/neurips/2019/about.neurips.2019.html:
```diff
@@ -97,13 +98,35 @@ <h3>Platinum</h3>
 		  <a target="_blank" href="https://www.google.com"><img width=500
```
- Please add missing sponsor Facebook under platinum.
- Deepmind should be listed under Silver, not Gold.

mpantoja314 replied:
I did the images to the nips folder

mpantoja314 commented on _includes/neurips/2019/about.neurips.2019.html:
```diff
@@ -26,8 +26,9 @@ <h3>Important Dates</h3>
-              <li>October 23rd, 2019: NeurIPS early registration deadline</li>
-              <li> November 11th, 2019: <a target='_blank' href="#">NeurIPS Child Care Registration deadline</a></li>
+              <li>October 23rd, 2019: NeurIPS early registration deadline</li> 
+	      <li>Nov 1st, 2019: Registration starts for LXAI<a target='_blank' href="https://www.eventbrite.com/publish?crumb=311f3f067b6a32&eid=77877199801">Workshop</a></li>
```
done

mpantoja314 commented on _includes/neurips/2019/about.neurips.2019.html:
```diff
@@ -26,8 +26,9 @@ <h3>Important Dates</h3>
-              <li> November 11th, 2019: <a target='_blank' href="#">NeurIPS Child Care Registration deadline</a></li>
+              <li>October 23rd, 2019: NeurIPS early registration deadline</li> 
+	      <li>Nov 1st, 2019: Registration starts for LXAI<a target='_blank' href="https://www.eventbrite.com/publish?crumb=311f3f067b6a32&eid=77877199801">Workshop</a></li>
+	      <li> November 11th, 2019: <a target='_blank' href="#">NeurIPS Child Care Registration deadline</a></li>
```
done

mpantoja314 commented:
I did move images from sponsors to correct folder
added the correct link for eventbrite and kids 
Corrected the sponsors list and added a tag for presenters but I still don't know the order of presentation so I'm leaving that one blank. If you want I can add some random number there until I get the correct one

mpantoja314 commented:
I did add the presenters order
I did put all logos under 
assets/img/NIPS/2019/sponsors
and the images for the presenters on
assets/img/NIPS/2019/speakers
